source/application/AMAppController.cpp

```diff
@@ -139,12 +139,28 @@ bool AMAppController::startup(){
 
 void AMAppController::shutdown()
 {
-	AMDatamanAppControllerForActions3::shutdown();
+	AMActionRunner3::releaseScanActionRunner();
+	AMActionRunner3::releaseWorkflow();
+
 	AMBeamline::releaseBl();
 	AMStorageRing::releaseStorageRing();
 	AMProcessVariableSupport::shutdownChannelAccess();
 }
 
+bool AMAppController::startupBeforeUserInterface()
```

Comment:
Did this code used to live in AMDatamanAppController? Why has it been moved here?